.github/workflows/docker.yml

```diff
@@ -62,19 +62,72 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-        # This should work without any of the project sources except compose.yaml
         with:
-          sparse-checkout: |
-            compose.yaml
-          # Disable cone mode so we only grab a specific file
-          # https://github.com/actions/checkout?tab=readme-ov-file#fetch-only-a-single-file
-          # https://git-scm.com/docs/git-sparse-checkout
-          sparse-checkout-cone-mode: false
+          path: checkout
+      - uses: dtolnay/rust-toolchain@stable
+      - run: cargo build --package divviup-cli --profile release
+        working-directory: checkout
+      - run: |
+          mkdir compose
+          cp checkout/target/release/divviup compose/
+          cp checkout/compose.yaml compose/
       - name: Compose
         id: compose
         run: docker compose up --wait --wait-timeout 120
+        working-directory: compose
+      - name: Demo script
+        id: demo-script
+        working-directory: compose
+        run: |
+          export DIVVIUP_API_URL=http://localhost:8080
+          export DIVVIUP_TOKEN=""
+
+          export DIVVIUP_ACCOUNT_ID=`./divviup account list | jq -r '.[0].id'`
+          printf 'account ID %s\n' $DIVVIUP_ACCOUNT_ID
+
+          AGGREGATOR_LIST=`./divviup aggregator list`
+          printf 'aggregator list %s\n' $AGGREGATOR_LIST
+
+          LEADER_ID=`echo $AGGREGATOR_LIST | jq -r '.[] |= select(.name == "leader") |.[0].id'`
+          printf 'leader ID %s\n' $LEADER_ID
+
+          HELPER_ID=`echo $AGGREGATOR_LIST | jq -r '.[] |= select(.name == "helper") |.[0].id'`
+          printf 'helper ID %s\n' $HELPER_ID
+
+          CRED_OUTPUT=`./divviup collector-credential generate --save`
+          printf 'collector credential %s\n' $CRED_OUTPUT
+
+          COLLECTOR_CREDENTIAL_PATH=${PWD}/`echo $CRED_OUTPUT | jq -r '.name' 2>/dev/null || echo ''`.json
+          printf 'collector credential path %s\n' $COLLECTOR_CREDENTIAL_PATH
+
+          COLLECTOR_CREDENTIAL_ID=`echo $CRED_OUTPUT | jq -r '.id' 2>/dev/null || echo ''`
+          printf 'collector credential ID %s\n' $COLLECTOR_CREDENTIAL_ID
+
+          TASK=`./divviup task create --name net-promoter-score \
+            --leader-aggregator-id $LEADER_ID --helper-aggregator-id $HELPER_ID \
+            --collector-credential-id $COLLECTOR_CREDENTIAL_ID \
+            --vdaf histogram --categorical-buckets 0,1,2,3,4,5,6,7,8,9,10 \
+            --min-batch-size 10 --max-batch-size 200 --time-precision 60sec`
```

Comment:
I think this is going to run afoul of our validation: https://github.com/divviup/divviup-api/blob/152f3526df79166a21b5edc683d6c9fbdfb20b1a/src/entity/task/new_task.rs#L30-L31

Reply:
It does. That's kind of annoying. I worry this test will take forever to run if we have to wait for 100ish reports to be aggregated.